src/Environment.php

```diff
@@ -66,6 +67,7 @@ class Environment
     private $runtimeLoaders = [];
     private $runtimes = [];
     private $optionsHash;
+    private $useYield;
```

Comment:
this must be included in the options hash to invalidate all the cache when switching to yielding mode.

Reply:
Good catch!